src/server/tests/jobsApplication.test.js

```diff
@@ -25,7 +26,10 @@ beforeAll(async() => {
 
 });
 
-
+afterAll(async() => {
+    // Delete the user from the database after running all the tests
+    await User.findByIdAndDelete(userId);
+});
```

Comment:
It's confusing to have 2 `afterAll` statements in a test file

Reply:
> It's confusing to have 2 `afterAll` statements in a test file

you are right, ill merge them in one! Good catch